addons/mail/models/mail_thread.py

```diff
@@ -1469,6 +1469,8 @@ def _message_add_suggested_recipient(self, result, partner=None, email=None, lan
             return result
         if partner and partner.id in [val[0] for val in result[self.ids[0]]]:  # already existing partner ID -> skip
             return result
+        if lang and not self.env['res.lang']._lang_get(lang):
```

Comment:
I would add a python comment, something like:

`# check if that language is correctly installed (and active) before using it`